lib/pbio/drv/sound/beep_sampled.c

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025 The Pybricks Authors
+
+// Converts beeps into an array of samples
+
+#include <pbdrv/config.h>
+
+#if PBDRV_CONFIG_SOUND_SAMPLED_ONLY
```

Comment:
```suggestion
#if PBDRV_CONFIG_SOUND_BEEP_SAMPLED
```
It would be preferable to make the config option match the file name.